index.js

```diff
@@ -154,33 +155,24 @@ module.exports = {
       { missingExports: false, unusedExports: true }
     ], */
 
-    // react
-    // TODO turn all rules to "error" eventually
-    "react/button-has-type": "warn",
-    "react/display-name": "warn",
+    // react, customization of rules
+    "react/button-has-type": "error",
     "react/jsx-boolean-value": ["error", "always"],
-    "react/jsx-no-comment-textnodes": "warn",
-    "react/jsx-no-target-blank": "warn",
     "react/jsx-pascal-case": "error",
-    "react/jsx-sort-default-props": "warn",
-    "react/jsx-sort-props": "warn",
-    "react/jsx-fragments": ["warn", "syntax"],
+    "react/jsx-sort-default-props": "error",
+    "react/jsx-fragments": "error",
     "react/jsx-curly-spacing": [
       "error",
       { when: "never", allowMultiline: true }
     ],
     "react/jsx-curly-brace-presence": [
       "error",
-      { props: "never", children: "never" }
+      { props: "never", children: "ignore" }
```

Comment:
Updates to the plugin made this rule noisy for children, so I disabled it in this specific case